src/main/java/fithelper/model/UserProfile.java

```diff
@@ -0,0 +1,184 @@
+//package fithelper.model;
```

Comment:
What is the usage of this class?

Reply:
It's a parallel class of FitHelper.java: FitHelper handles entry data storage, whereas UserProfile handles profile data storage.